class MyError(Exception):
    pass

raise MyError("This is my error")
